recipes/crc32-cpp/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "crc32c" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}-cpp
```

Comment:
Why a `-cpp` name?

Reply:
We have a `crc32c` Python package on conda-forge already and this is the C++ implementation of `crc32c`.

Reply:
Ah. okay. How about `libcrc32c`?

Reply:
Done and CI passed.